### ISSUE TITLE
Create site-wide notice

### ISSUE THREAD

**Issue by ianb** [mozilla-services/screenshots#5248]
We want to have a message on all pages (except the homepage). Probably some top bar? It would link to the future page (#5247)

**Comment by 6a68** [comment]
Yeah, we can just add a new message to the AdBanner component, and add that case to the PromotionStrategy to show on all pages. Very little engineering effort needed.

**Comment by 6a68** [comment]
Ah, but as with #5247, any strings would need to be exposed to localizers, so we might want to be thoughtful in when the PR is landed

**Comment by johngruen** [comment]
We dicided not to display this on the 66-tied server release but to do it shortly thereafter.
